package.json

```diff
@@ -28,7 +28,7 @@
     "style:compile": "node-sass -q --output-style expanded --precision 5 src/ --output dist/",
     "style:postCSS": "postcss --config config/postcss.config.js  --replace dist/*.css",
     "style:remove": "if [ -d dist/ ]; then rm -rf dist/; fi",
-    "test": "npm run test:linux-fix && server-test start:playground http-get://localhost:3030 test:visual",
+    "test": "npm run storybook:visual",
     "test:linux-fix": "cp -rf scripts/runDocker.js node_modules/backstopjs/core/util/",
```

Comment:
Can we replace all these test commands with the storybook testing ones and delete the storybook testing ones to clean this up a bit?

Reply:
should i do it in this PR?

Reply:
done